### PR TITLE
remove terms of service

### LINKED ISSUE #73: Terms of service link
The terms of service link on this page, https://frontend-v2-test.apps.silver.devops.gov.bc.ca/about/orgbook-api, links to The Apache License info here; https://github.com/bcgov/orgbook-api/blob/main/LICENSE.

It should link to an actual terms if service page, which describe the terms of service for the API.

### PR DESCRIPTION
**Pull request by wadeking98**
Resolves: #73 remove terms of service link
Signed-off-by: wadeking98 <wkingnumber2@gmail.com>

### CODE REVIEW

**Review by WadeBarnes — COMMENTED** [review]
I think we need to remove the phrase altogether.  @swcurran, would you agree?

**Comment by swcurran** [comment]
Yes -- right now we don't have, so nothing to follow.

**Review by amanji — DISMISSED** [review]

File: src/docs/orgbook_api.md [hunk 1]
```diff
@@ -12,6 +12,6 @@ Anyone is able to use OrgBook BC data in their own services. Rather than needing
 
 Developers can review the [API specification](http://orgbook.gov.bc.ca/api/) and the [API documentation](https://github.com/bcgov/orgbook-bc-api-docs). There is also an [API demo](https://github.com/bcgov/orgbook-bc-api-docs/tree/main/demo) for developers to see it in action.
 
-Any use of the API is subject to the [terms of service](https://github.com/bcgov/orgbook-bc-api-docs/blob/main/LICENSE).
+Any use of the API is subject to the terms of service.
```
Comment (WadeBarnes) [inline src/docs/orgbook_api.md]: @wadeking98, Please remove this line altogether.

**Review by WadeBarnes — CHANGES_REQUESTED** [review]

File: src/docs/orgbook_api.md (hunk 1 above)
Comment (wadeking98) [reply src/docs/orgbook_api.md]: line removed

**Review by amanji — APPROVED** [review]

**Review by WadeBarnes — APPROVED** [review]

**Pull request merged**